travis-ymls/jung-algorithms.travis.yml

```diff
@@ -0,0 +1,51 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : jung-algorithms
+# Source Repo         : https://github.com/jrtom/jung
+# Travis Job Link     : https://travis-ci.com/github/santosh653/jung
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+# Per https://docs.travis-ci.com/user/languages/java
+arch:
+  - amd64
+  - ppc64le
+sudo: false
+
+language: java
+
+dist: trusty
+
+# openjdk8 is currently used for tools/deploy_snapshots.sh; if we stop using that JDK here,
+# change the JDK version used in that script
+jdk:
+  - oraclejdk8
```

Comment:
Exclude `oraclejdk8` job on power. @santosh653

Reply:
@Siddhesh-Ghadi I have excluded oraclejdk8 on power & squashed the commits.